app/lib/pages/settings/changelog/change.dart

```diff
@@ -50,21 +50,54 @@ class LineStringList with IterableMixin<String> {
 }
 
 /// An Object which lets one compare different App-Versions.
-/// The Version must be three version number seperated by docs (e.g. "1.3.1").
+///
+/// The Version must be three version number separated by docs (e.g. "1.3.1").
 class Version {
   final String name;
-  final versionRegEx = RegExp(r'^(\d+\.)(\d+\.)(\d+)$');
+  static final _versionRegEx = RegExp(r'^(\d+\.)(\d+\.)(\d+)');
 
-  Version({@required String name})
-      : name = name.replaceAll(
-            RegExp(r'\W*(-dev)\W*'), ""); // Removing -dev of version
+  /// The major number of the version, incremented when making breaking changes.
+  final int major;
+
+  /// The minor number of the version, incremented when adding new functionality
+  /// in a backwards-compatible manner.
+  final int minor;
+
+  /// The patch number of the version, incremented when making
+  /// backwards-compatible bug fixes.
+  final int patch;
+
```

Comment:
Tbh we don't use the numbers like this. As it is just an app, not a library the numbers are just to show people how big of a change we made I guess. What even would be a breaking change? If the old versions don't work anymore? But I guess we would always make changes where old versions still work at least for some time.
I think that these comments might just cause more confusion as they state something that is not true in my opinion.

Reply:
Yes, you're right, I removed them 👍